Resources/Prototypes/_LostParadise/Entities/Structures/Wallmounts/newyear.yml

```
@@ -77,10 +77,10 @@
   name: light bulbs
   description: Hanging garland, not the best source of light
   components:
-  - type: PointLight
-    radius: 2.0
-    energy: 2.5
-    color: "#D6001C"
+  #- type: PointLight    #лагает страшно
```

yamllint (1.35.1)

[warning] missing starting space in comment

(comments)